[BUG]Missing newly added template in 8.2.0 are missing under template doc (#1961)
elastic/security-docs: docs/events/timeline-templates.asciidoc

NOTE: For information on how to add Timeline templates to rules, see
<<create-rule-ui>>.

> When you load {es-sec} prebuilt rules, some prebuilt Timeline templates are also loaded. You can attach these templates to detection rules. *Generic* templates use broad KQL queries to retrieve event data, and *Comprehensive* templates use detailed KQL queries for additional information. The following templates are loaded:

Reviewer: ```suggestion
When you load {es-sec} prebuilt rules, {es-sec} also loads a selection of prebuilt Timeline templates. You can attach these templates to detection rules. *Generic* templates use broad KQL queries to retrieve event data, and *Comprehensive* templates use detailed KQL queries to retrieve additional information. The following prebuilt templates appear by default:
```

> When you load {es-sec} prebuilt rules, {es-sec} also loads a selection of prebuilt Timeline templates. You can attach these templates to detection rules. *Generic* templates use broad KQL queries to retrieve event data, and *Comprehensive* templates use detailed KQL queries to retrieve additional information. The following prebuilt templates appear by default:

Reviewer 2: ```suggestion
When you load {elastic-sec} prebuilt rules, {elastic-sec} also loads a selection of prebuilt Timeline templates, which you can attach to detection rules. *Generic* templates use broad KQL queries to retrieve event data, and *Comprehensive* templates use detailed KQL queries to retrieve additional information. The following prebuilt templates appear by default:
```

> * *Generic Endpoint Timeline*: Investigate Elastic Endpoint detection alerts.

Reviewer 2: ```suggestion
* *Generic Endpoint Timeline*: Investigate {elastic-endpoint} detection alerts.
```
